src/test/java/org/jabref/gui/entryeditor/SourceTabTest.java

```diff
@@ -40,7 +48,16 @@
     public void onStart(Stage stage) {
         area = new CodeArea();
         area.appendText("some example\n text to go here\n across a couple of \n lines....");
-        sourceTab = new SourceTab(new BibDatabaseContext(), new CountingUndoManager(), new LatexFieldFormatterPreferences(), mock(ImportFormatPreferences.class), new DummyFileUpdateMonitor(), mock(DialogService.class), mock(StateManager.class));
+        StateManager stateManager = mock(StateManager.class);
+        when(stateManager.activeSearchQueryProperty()).thenReturn(OptionalObjectProperty.empty());
+        Globals.prefs = mock(JabRefPreferences.class);
```

Comment:
Is it really that complicated to extract the preferences and replace `Globals.prefs` by constructor arguments?

Reply:
You're right. Fixed.